connector_ecommerce/migrations/14.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,11 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    try:
+        rule = env.ref('connector_ecommerce.excep_product_has_checkpoint')
+    except ValueError:
+        rule = None
```

Comment:
```suggestion
    rule = env.ref(
        "connector_ecommerce.excep_product_has_checkpoint",
        raise_if_not_found=False,
    )
```

Reply:
Thanks @ivantodorovich, the suggestion has been applied.